languages/csharp/exercises/concept/numbers-floating-point/.docs/after.md

```diff
@@ -1,14 +1,16 @@
 # After
 
-- See the [characteristics of the floating-point types table][docs-microsoft.com-characteristics-of-the-floating-point-types] for an overview of the precision, approximate range and size of the three floating-point types.
+There are three floating-point types in C#: `float`, `double` and `decimal`. The most commonly used type is `double`, whereas `decimal` is normally used when working with monetary data.
 
-- Converting between numeric types is sometimes [automatic (implicit)][docs-microsoft.com-implicit-numeric-conversion], but at other times [manual (explicit)][docs-microsoft.com-explicit-numeric-conversion].
+Each floating-point type has its own [precision, approximate range and size][docs-microsoft.com-characteristics-of-the-floating-point-types].
 
-- Be careful when checking the values of floating-point types for equality, as values that can appear to represent the same value could actually be different. See [this article][docs.microsoft.com_precision-in-comparisons] for more information.
+Some conversions between floating point types are [automatic (implicit)][docs-microsoft.com-implicit-numeric-conversion], but others are [manual (explicit)][docs-microsoft.com-explicit-numeric-conversion].
 
-- You can find a short introduction to floating-point numbers at [0.30000000000000004.com][0.30000000000000004.com].
+Always be careful when checking the values of floating-point types for equality, as values that can appear to represent the same value could actually be different. See [this article][docs.microsoft.com_precision-in-comparisons] for more information.
 
-- The [Float Toy page][evanw.github.io-float-toy] has a nice, graphical explanation how a floating-point numbers' bits are converted to an actual floating-point value.
+You can find a short introduction to floating-point numbers at [0.30000000000000004.com][0.30000000000000004.com]. The [Float Toy page][evanw.github.io-float-toy] has a nice, graphical explanation how a floating-point numbers' bits are converted to an actual floating-point value.
```

Comment:
question: I noticed that does not look like a markdown link and `evanw.github.io-float-toy` is not a valid URL. Is there a tool that converts this to a valid URL somewhere?